debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -753,6 +819,16 @@ public static SchemaRefreshMode parse(String value) {
                     + "'json' represents values as string-ified JSON (default)"
                     + "'map' represents values as a key/value map");
 
+    public static final Field BINARY_HANDLING_MODE = Field.create("binary.handling.mode")
```

Comment:
See above, should be pulled up. Note I'm fine with only exposing this option for Postgres initially, to keep scope limited.

Reply:
DONE